website/blog/2023-11-11-AgentEval/index.mdx

```diff
@@ -0,0 +1,144 @@
+
+---
+title:  AgentEval: Assessing Task Utility for LLM-powered Applications
+authors: julianakiseleva, narabzad
+tags: [LLM, GPT, evaluation, task utility]
+---
+
+![Fig.1: An verification framework](img/agenteval-CQ.png)
+
+**TL;DR:**
+* We introduce a notion of what is task utility in LLM-powered applications

```

Comment:
We introduce the notion of task utility for evaluating LLM-powered applications.